tests/test_app.py

```
@@ -0,0 +1,233 @@
+    mock_api_global.provide_feedback.assert_called_once_with(message_content, feedback_label)
+
+    # Account for HTML escaping of single quotes
+    escaped_label = feedback_label.replace("'", "&#39;")
+    # The message content itself in the flash message is not escaped by f-string, but by Jinja2.
+    # However, the part of the message we construct for comparison here (message_content[:50])
+    # should also have its single quotes escaped if we are comparing against the rendered HTML.
+    # For "A message that was classified", there are no single quotes.
+    # If message_content had single quotes, e.g. "It's a message", it would become "It&#39;s a message"
+    escaped_message_snippet = message_content[:50].replace("'", "&#39;")
+
+    expected_flash_text = f"Feedback ('{escaped_label}') submitted successfully for: '{escaped_message_snippet}..."
```

Ruff (0.11.9)

Local variable `expected_flash_text` is assigned to but never used

Remove assignment to unused variable `expected_flash_text`

(F841)